tests/e2e/test_pvc_creation_performance.py

```diff
@@ -43,9 +46,50 @@ def test_pvc_creation_measurement_performance(self, teardown_factory):
 
         pvc_obj = create_pvc(sc_name=self.sc_obj.name, size=self.pvc_size)
         teardown_factory(pvc_obj)
-        create_time = measure_pvc_creation_time('CephBlockPool', pvc_obj.name)
+        create_time = measure_pvc_creation_time(self.interface, pvc_obj.name)
         if create_time > 1:
             raise AssertionError(
                 f"PVC creation time is {create_time} and greater than 1 second"
             )
         logging.info("PVC creation took less than a 1 second")
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    @polarion_id('OCS-1620')
+    @bugzilla('1741612')
+    def test_multiple_pvc_creation_measurement_performance(
+        self, teardown_factory
+    ):
+        """
+        Measuring PVC creation time of 120 PVCs in 60 seconds
+        """
+        number_of_pvcs = 120
+        log.info('Start creating new 120 PVCs')
+
+        pvc_objs = create_multiple_pvcs(
+            sc_name=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            number_of_pvc=number_of_pvcs,
+            size=self.pvc_size,
+            wait=False
+        )
+        for pvc_obj in pvc_objs:
+            teardown_factory(pvc_obj)
+        for pvc_obj in pvc_objs:
```

Comment:
If the PVCs don't get bound for some reason, this test case is going to take 120 * 60 seconds(2 hours!) before marking it as failed. we should arrive at a better way to check for bound status for bulk creations.

Reply:
I don't have a smart solution for it right now.
That's a limitation

Reply:
May be using threads would help to reduce the time.

Reply:
not really,
If I'm creating 120 PVCs without waiting for each one, I can assume that when finishing the creation of the last PVC, many of the first PVCs already bounded so the "wait" will return immediately.
When reaching to the first PVC the is still Pending, I will wait for the Bound status and in that same time, few more PVCs will reach Bound state.

So threads are not really helpful here cause the operation is not in sequence